sql.txt

```diff
@@ -1,14 +1,28 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 9 OFFSET 10;
+
```

Comment:
Almost! For the `LIMIT` you want to use 10 instead of 9 because the range 11 through 20 is a total of 10 entries. `20-11 + 1 = 10`